src/economy.cpp

```diff
@@ -984,6 +984,9 @@ Money GetTransportedGoodsIncome(uint num_pieces, uint dist, uint16_t transit_per
 		return 0;
 	}
 
+	/* Scale transit periods according to the game setting. NewGRFs callbacks operate on scaled value for compatibility. */
```

Comment:
Why mention NewGRFs when they don't seem to be handled any differently? If that is done here, don't mention it. If it's done elsewhere... I can't see it in the changed files. 😉

Reply:
It was supposed to explain why is scaling done before the callback and not after

Reply:
```suggestion
	/* Scale transit periods according to the game setting. We pass this scaled value to the NewGRF callback. */
```

Reply:
"We also pass this"